app/navigation/JourneyNavigator.scala

```diff
@@ -44,17 +46,23 @@ class JourneyNavigator @Inject()() extends Navigator {
 
     // Income from property, pensions and investments
     case RentalIncomePage                    => ua => controllers.propertypensionsinvestments.routes.PensionsController.onPageLoad(NormalMode, ua.taxYear)
-    case PensionsPage                        => ua => controllers.propertypensionsinvestments.routes.UkInsuranceGainsController.onPageLoad(NormalMode, ua.taxYear)
+    case PensionsPage                        => ua => controllers.propertypensionsinvestments.routes.UkInsuranceGainsController.
+      onPageLoad(NormalMode, ua.taxYear)
     case UkInsuranceGainsPage                => ua => controllers.propertypensionsinvestments.routes.UkInterestController.onPageLoad(NormalMode, ua.taxYear)
-    case UkInterestPage                      => ua => controllers.propertypensionsinvestments.routes.UkDividendsSharesLoansController.onPageLoad(NormalMode, ua.taxYear)
+    case UkInterestPage                      => ua => controllers.propertypensionsinvestments.routes.UkDividendsSharesLoansController.
+      onPageLoad(NormalMode, ua.taxYear)
     case UkDividendsSharesLoansPage          => ua => routes.AddSectionsController.onPageLoad(ua.taxYear)
 
-
     // Payments into pensions
     case PaymentsIntoPensionsPage             => ua => routes.AddSectionsController.onPageLoad(ua.taxYear)
 
+    //Income from work and taxable state benefits section
+    case AboutYourWorkRadioPage               => ua => controllers.workandbenefits.routes.ConstructionIndustrySchemeController.
+      onPageLoad(NormalMode, ua.taxYear)
+    case AboutYourWorkPage => aboutYourWorkRoute
+    case ConstructionIndustrySchemePage       => ua => controllers.workandbenefits.routes.JobseekersAllowanceController.onPageLoad(NormalMode, ua.taxYear)
+    case JobseekersAllowancePage              => ua => routes.AddSectionsController.onPageLoad(ua.taxYear)
```

Comment:
I think it may be better if this section was put into the file in the same order as the sections appear on the "Add sections" page, so this section would go below "About You".   Also for the PrivateBetaNavigator and both specs.